fedimint-server/src/consensus/api.rs

```diff
@@ -625,7 +625,7 @@ pub fn server_endpoints() -> Vec<ApiEndpoint<ConsensusApi>> {
         },
         api_endpoint! {
             SHUTDOWN_ENDPOINT,
-            ApiVersion::new(0, 0),
+            ApiVersion::new(0, 3),
```

Comment:
@bradleystachurski Did we even increment currently supported to 0.3? Last time I checked it was 0.2.

Reply:
Yep, we're on 0.3 since this is pointed to master 👍 

https://github.com/fedimint/fedimint/blob/6c888ae86cdabe52840f381dc241501b1229a052/fedimint-server/src/config/mod.rs#L207

I'm putting together the backport PR that also bumps releases/v0.3 from 0.2 to 0.3